src/operation.ts

```diff
@@ -326,19 +326,34 @@ export default class Operation {
    * Get an `operationId` for this operation. If one is not present (it's not required by the spec!) a hash of the path
    * and method will be returned instead.
    *
+   * @param shouldCamelCase Generate a JS method-friendly operation ID when an `operationId` isn't present.
    */
-  getOperationId(): string {
+  getOperationId(shouldCamelCase = false): string {
```

Comment:
Any alternative API design here to avoid the [boolean trap](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap)?

Maybe `getOperationId({ camelCase: true })` or something?